gerrymander/git.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2014 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import subprocess
+import urlparse
+
+def get_git_config(key):
+    '''Read a git configuration value use "git config --get ..."'''
+    val = subprocess.check_output([
+        'git', 'config', '--get', key])
```

Comment:
We need to catch subprocess.CalledProcessError  and return None, otherwise your check 'if not url' in get_remote_info will never run